sdk/communication/azure-communication-identity/tests/test_communication_identity_client.py

```diff
@@ -69,58 +66,57 @@ def test_create_user_and_token(self, communication_livetest_dynamic_connection_s
 
         assert user.properties.get('id') is not None
         assert token_response.token is not None
-        
-    @parameterized.expand(
-        [
-            ("min_valid_hours", 1),
-            ("max_valid_hours", 24),
-        ]
-    ) 
-    def test_create_user_and_token_with_valid_custom_expirations(self, _, valid_hours):
+
+    @CommunicationPreparer()
+    @pytest.mark.parametrize("_, value", [("min_valid_hours", 1), ("max_valid_hours", 24)])
```

Comment:
Does this `pytest.mark.parametrize` use satisfy your test requirements, or do you still need a workaround for using `parameterized.expand`? I assumed your tests were sorted out once I saw your PR, but I just want to be sure.

In general I would recommend using pytest's `parametrize` decorator because it avoids a third-party dependency on `parameterized`, but it depends on the situation.

Reply:
Yes, the approach I'm using in this PR satisfies our requirements. I also think that reducing the number of external dependencies is good.